src/config.py

```diff
@@ -7,4 +7,10 @@
 TOKEN = os.environ.get("TOKEN")  # required
 DEBUG = os.environ.get("DEBUG", False)
 
+ROLE_IDS = [
+  ("Webgroup", "1166751688598761583"),
+  ("Gamez", "1089204642241581139"),
+  ("Croomer", "1172696659097047050"),

```

Comment:
I don't mind converting the ids to strings, but do this for all ids for consistency (i.e. the lobby channel id) and ensure they still work as intended.

Reply:
it error when i tried ints 😂. hikari.CommandChoice insists on value being str or some other inferable type

I'll convert the rest ←⁠(⁠>⁠▽⁠<⁠)⁠ﾉ.  😪